new_lineage_panel/src/App.tsx

```diff
@@ -173,13 +182,13 @@ function App() {
       const existingNode = _flow.getNode(node.table);
       if (existingNode) {
         setSelectedTable(node.table);
-        const [nodes, edges] = highlightTableConnections(
-          _flow.getNodes(),
-          _flow.getEdges(),
-          node.table
-        );
-        _flow.setNodes(nodes);
-        _flow.setEdges(edges);
+        let nodes = _flow.getNodes();
```

Comment:
Better to avoid mutable variables and extract this code to separate function which could return nodes and edges

Reply:
Rest of the code is not design for this. Will redesign it later in future PR.